openslides_backend/action/actions/option/update.py

```diff
@@ -149,3 +169,14 @@ def _get_vote_id(
             if item["value"] == search_value:
                 return key
         return None
+
+
+def check_state_change(instance: Dict[str, Any], poll: Dict[str, Any]) -> bool:
+    if poll.get("type") != Poll.TYPE_ANALOG:
+        return False
+    if poll.get("state") != Poll.STATE_CREATED:
```

Comment:
This check is not right. If a poll is *finished* and a user updates the poll and sets `publish_immediately`, the poll must be set to published.

Same for the other two functions.

Reply:
Update the publish_immediately logic.

Reply:
This has to be done for all three methods.

Reply:
What do you mean? I have updated that for all three actions.